protocol-units/execution/maptos/opt-executor/src/background/transaction_pipe.rs

```diff
@@ -481,6 +450,52 @@ mod tests {
 		(context, transaction_pipe, tempdir)
 	}
 
+	#[derive(Clone)]
+	pub struct TxPipeTestDaSequencerClient {
+		pub received_tx: Arc<Mutex<Vec<MvTransaction>>>,
+	}
+
+	impl TxPipeTestDaSequencerClient {
```

Comment:
If we're implemented a test trait, I don't think we want to keep this in non-feature flagged production code.

Reply:
It's in the Test part, so it's not in production code?

Reply:
Ah sorry I misread in GH UI.